Source/Blazorise.Tailwind/Styles/components/_form.scss

```diff
@@ -0,0 +1,4 @@
+﻿.label-required:before {
```

Comment:
What do you think of doing it in the tailwind style? https://tailwindcss.com/docs/content

Reply:
Is that good for overriding? 
Since you have more experience would it be something like : `before:content-['*']` and how do we color style it?

Reply:
I believe you can still use `var` for colors.

Reply:
ps. https://stackoverflow.com/questions/64872861/how-to-use-css-variables-with-tailwind-css

Reply:
before:content-['*'] before:[color:red]

Reply:
However it doesn't seem like it supports the blank space do you know anything about it? `before:content-['* ']`

Reply:
Seems decent enough?
![image](https://github.com/Megabit/Blazorise/assets/22283161/cb7b102b-f094-43a0-8ca0-6fd4a2354cb5)

Reply:
Shouldn't it need to be after the label?

![image](https://github.com/Megabit/Blazorise/assets/900302/849cf16c-ac31-40d5-bbd4-7075c29c5085)

Reply:
Well I don't know from an ux perspective... . I think by being the first thing you can tell the required fields right away. While if they are at the end it can be something that you don't notice right away? The bigger the label is the further it gets pushed.

Reply:
But yes I have seen both ways of doing it.

Reply:
> Well I don't know from an ux perspective... . I think by being the first thing you can tell the required fields right away. While if they are at the end it can be something that you don't notice right away? The bigger the label is the further it gets pushed.

It seems from a UX perspective it is better to have on the end

https://uxpickle.com/required-fields-in-forms/
https://uxdesign.cc/form-field-required-vs-optional-9b4d7cdbf400

Reply:
They don't seem to talk about the design decision **before vs after** do they ? 
We can't just base ourselves in some articles...

I see arguments for both sides...
![image](https://github.com/Megabit/Blazorise/assets/22283161/a7d61bf4-9903-491d-b2d9-10a1ce6150b5)

Anyway as long as its consistent.. Would you prefer to switch to after then?

Reply:
W3C (Altough it seems old) In here they deem important to define the meaning of it. And in the example its after the table.

https://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/H90
![image](https://github.com/Megabit/Blazorise/assets/22283161/9fb2b158-6352-440d-91b8-12fb3500299a)

Reply:
(plus they have recommended markup with abbr that we don't comply, nor do I think we need to bother? :D:D)

Reply:
🤔well, we have in plan to support AODA, so I think it is worth doing it with `abbr`.

Reply:
oh well... There goes all the dev haha. Alright Let's follow these acessibility guidelines... but if you want to support some other guidelines, AODA? Then we should read those up?
Is it this? https://aoda.ca/ There does not seem to be web related guidelines?

Reply:
This seems newer standard definitions
https://www.w3.org/WAI/tutorials/forms/validation/
![image](https://github.com/Megabit/Blazorise/assets/22283161/9d043a21-ed11-4221-9ccf-f4fe49722d48)
I'm not sure this is a rabbit hole we want to go into just yet...?

Reply:
haha, leave it for some in the future. Better to go with the other 1.3 tasks.

Reply:
Ye, I would just do a release focused on this. It seems like we need to do alot of research.
We can however move the * to after for now.